Lib/test/test_posix.py

```diff
@@ -1522,7 +1522,7 @@ def test_no_such_executable(self):
             pid = self.spawn_func(no_such_executable,
                                   [no_such_executable],
                                   os.environ)
-        except FileNotFoundError as exc:
+        except (FileNotFoundError, PermissionError) as exc:
```

Comment:
Maybe add a comment mentioning bpo-35794 to explain why PermissionError is expected.